modules/local/atacreads/getscale.nf

```diff
@@ -2,30 +2,22 @@ process GET_SCALE {
     tag "${meta.id}"
     label 'process_low'
 
-    conda "anaconda::gawk=5.1.0"
-    container "${ workflow.containerEngine == 'singularity' &&
-                    !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/gawk:5.1.0' :
-        'quay.io/biocontainers/gawk:5.1.0' }"
-
     input:
     tuple val(meta), val(counts)
     val min_counts
 
     output:
     tuple val(meta), val(scale)  , emit: scale
-    path "versions.yml"                               , emit: versions
+    path "versions.yml"          , emit: versions
 
     script:
     def software = "awk"
     scale = min_counts/counts.toInteger()
     def prefix   = task.ext.prefix ? "${meta.id}${task.ext.prefix}" : "${meta.id}"
```

Comment:
Don't use this anymore.
I'd also say that typically you just put 
`def prefix   = task.ext.prefix ? "${task.ext.prefix}" : "${meta.id}"`
and then people can use the meta.id or not when they define the prefix.

Reply:
Got it.